src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -75,6 +75,16 @@ SolverBase::SolverBase( string const & name,
     setRestartFlags( RestartFlags::WRITE_AND_READ ).
     setDescription( "Initial time-step value required by the solver to the event manager." );
 
+  registerWrapper( viewKeyStruct::computeStatisticsString(), &m_computeStatistics ).
+    setApplyDefaultValue( 0 ). // do nothing by default
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Flag indicating whether statistics are computed or not" );
```

Comment:
In the new version of the PR, I would like to use `TimeHistory` to output the statistics over time (i.e., no output to terminal, no output to vtk). So, I think it would be better not to use `plotLevel` or `logLevel` to control the behavior from the XML file.